solutions/devicesimulation/single-vm/setup.sh

```diff
@@ -141,7 +167,9 @@ echo "  aad : {"                                      >> ${WEBUICONFIG_UNSAFE}
 echo "    tenant: '${PCS_WEBUI_AUTH_AAD_TENANT}',"    >> ${WEBUICONFIG_UNSAFE}
 echo "    appId: '${PCS_WEBUI_AUTH_AAD_APPID}',"      >> ${WEBUICONFIG_UNSAFE}
 echo "    instance: '${PCS_WEBUI_AUTH_AAD_INSTANCE}'" >> ${WEBUICONFIG_UNSAFE}
-echo "  }"                                            >> ${WEBUICONFIG_UNSAFE}
+echo "  },"                                           >> ${WEBUICONFIG_UNSAFE}
+echo "  maxDevicesPerSimulation: 10000000,"           >> ${WEBUICONFIG_UNSAFE}
+echo "  minTelemetryInterval: 10000"                  >> ${WEBUICONFIG_UNSAFE}
```

Comment:
this change and the one at lines 157-159 should be in both templates - also there is a related change in the UI to use these settings

Reply:
updated